### ISSUE TITLE
Geographic data returned by the get_geo_data function does not match data returned by get_jts_data

### ISSUE THREAD

**Issue by Robinlovelace** [ITSLeeds/jts#5]
As we saw in live demo on ActDev project. Sorry for no reproducible example. Also thinking at some point we could look at the relationship between JTS data and safety data in the SaferActive project but that's for another day and probably another year!

**Comment by Robinlovelace** [comment]
Reproducible example:

```r
remotes::install_github("itsleeds/jts")
lads = jts::get_jts_data(table = "jts0401", output_format = "sf")
plot(lads)
```

Result shown below (note gaps!)

<details>

``` r
remotes::install_github("itsleeds/jts")
#> Using github PAT from envvar GITHUB_PAT
#> Skipping install of 'jts' from a github remote, the SHA1 (36bc5c5e) has not changed since last install.
#>   Use `force = TRUE` to force installation
lads = jts::get_jts_data(table = "jts0401", output_format = "sf")
#> 134 files found in path
#> Joining, by = "LA_Code"
plot(lads)
#> Warning: plotting the first 9 out of 94 attributes; use max.plot = 94 to plot
#> all
```

![](https://i.imgur.com/V60FiMZ.png)

<sup>Created on 2020-12-20 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

</details>

I think this can be fixed by changing this line of code to look for an older (2011?) definition of LADs:

https://github.com/ITSLeeds/jts/blob/36bc5c5e741e7d979c03c9c969cdb60e9a53c2cd/R/accessibility.R#L253

I think this data works: https://geoportal.statistics.gov.uk/datasets/fef73aeaf13c417dadf2fc99abcf8eef_0

Available as a json here (more generalised data may be preferable): https://geoportal.statistics.gov.uk/datasets/fef73aeaf13c417dadf2fc99abcf8eef_0.geojson

**Comment by Robinlovelace** [comment]
Equivalent code for LSOAs, which seems to work fine :tada: 

``` r
remotes::install_github("itsleeds/jts")
#> Using github PAT from envvar GITHUB_PAT
#> Skipping install of 'jts' from a github remote, the SHA1 (36bc5c5e) has not changed since last install.
#>   Use `force = TRUE` to force installation

lads = jts::get_jts_data(table = "jts0501", output_format = "sf")
#> 134 files found in path
#> Joining, by = "LSOA_code"
plot(lads)
#> Warning: plotting the first 10 out of 95 attributes; use max.plot = 95 to plot
#> all
```

![](https://i.imgur.com/p2poRfu.png)

<sup>Created on 2020-12-20 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by Robinlovelace** [comment]
Is this issue fixed after #7 was merged @joeytalbot ? We can close this if so.
